pyproject.toml

```diff
@@ -6,11 +6,12 @@ authors = []
 
 [tool.poetry.dependencies]
 python = "^3.7"
-flask = "^1.1.2"
+flask = "^1.1.4"
 python-dotenv = "^0.18.0"
 requests = "^2.25.1"
 pytest = "^6.2.4"
 selenium = "^3.141.0"
```

Comment:
To actually get any benefit from `poetry install --no-dev`, you should move pytest and selenium down to the "dev-dependencies" section.